client/components/data/document-head/index.jsx

```diff
@@ -75,29 +80,18 @@ class DocumentHead extends Component {
 DocumentHead.propTypes = {
 	title: PropTypes.string,
 	description: PropTypes.string,
-	unreadCount: PropTypes.number,
+	unreadCount: PropTypes.string, // E.g. '40+'
```

Comment:
Maybe we could accept either and coerce it into a string? Feels more natural to pass a number.

```js
unreadCount: PropTypes.oneOfType( [
	PropTypes.number,
	PropTypes.string
] );

// ...

if ( 'unreadCount' in this.props ) {
	this.props.setUnreadCount( String( this.props.unreadCount ) );
}
```